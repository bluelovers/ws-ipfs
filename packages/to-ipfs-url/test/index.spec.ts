import { toURL, toLink, EnumIPFSLinkType } from '../index';

describe(`describe`, () =>
{

	test(`toURL`, () =>
	{

		let actual = toURL(`Qmdbkxdh8tUA7zcnmcU6Nu7B7bNtubYTBjUHes799tgu83`, '001.png');
		let expected = "https://ipfs.io/ipfs/Qmdbkxdh8tUA7zcnmcU6Nu7B7bNtubYTBjUHes799tgu83?filename=001.png";


		expect(actual).toMatchSnapshot();
		expect(actual.toString()).toStrictEqual(expected);

	});

	test(`toLink:ipfs`, () =>
	{

		let actual = toLink(`Qmdbkxdh8tUA7zcnmcU6Nu7B7bNtubYTBjUHes799tgu83`, {
			type: EnumIPFSLinkType.ipfs,
		});
		let expected = "https://ipfs.io/ipfs/Qmdbkxdh8tUA7zcnmcU6Nu7B7bNtubYTBjUHes799tgu83";


		expect(actual).toMatchSnapshot();
		expect(actual.toString()).toStrictEqual(expected);

	});

})
