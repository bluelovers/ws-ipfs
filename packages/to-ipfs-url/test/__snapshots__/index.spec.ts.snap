// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`describe toLink:ipfs 1`] = `"https://ipfs.io/ipfs/Qmdbkxdh8tUA7zcnmcU6Nu7B7bNtubYTBjUHes799tgu83"`;

exports[`describe toURL 1`] = `"https://ipfs.io/ipfs/Qmdbkxdh8tUA7zcnmcU6Nu7B7bNtubYTBjUHes799tgu83?filename=001.png"`;
